src/init.cpp

```diff
@@ -1384,7 +1383,7 @@ bool AppInitMain(InitInterfaces& interfaces)
         for (const std::string& snet : gArgs.GetArgs("-onlynet")) {
             enum Network net = ParseNetwork(snet);
             if (net == NET_UNROUTABLE)
-                return InitError(strprintf(_("Unknown network specified in -onlynet: '%s'"), snet));
+                return InitError(strprintf(_("Unknown network specified in %s: '%s'"), "-onlynet", snet));
```

Comment:
bodies of an if statement should be on the same line or in brackets, imo this could be fixed in this PR